src/converters/piral-ng/src/converter.ts

```diff
@@ -3,6 +3,7 @@ import type { ForeignComponent, BaseComponentProps } from 'piral-core';
 import { createExtension } from './extension';
 import { enqueue } from './queue';
 import { bootstrap, NgModuleInt } from './bootstrap';
+import { NgOptions } from '.';
```

Comment:
Can we use `./types` for the import path?

Reply:
Damn auto imports ;), thanks for finding it. Fixed